articles/TOC.yml

```diff
@@ -29,50 +27,40 @@
     - name: Q# Jupyter notebooks
       href: install-guide/qjupyter.md
   - name: Update the QDK
-    href: install-guide/update.md  
-- name: Quickstarts
+    href: install-guide/update.md
+  - name: How to create a Q# project
+    href: how-to-guides/create-qsharp-project.md
+- name: Q# learning resources
   items:
-  - name: Quantum random number generator
-    href: quickstarts/qrng.md
-  - name: Quantum basics with Q#
-    href: quickstart.md
-  - name: Implement Grover's search algorithm
-    href: quickstarts/search.md
-- name: Concepts
+  - name: Quickstarts
+    items:
+    - name: Quantum random number generator
+      href: quickstarts/qrng.md
+    - name: Exploring entanglement with Q#
+      href: quickstart.md
+    - name: "Implement Grover's search algorithm"
+      href: quickstarts/search.md
+  - name: Learn Q# with the Quantum Katas
+    href: intro-to-katas.md
+  - name: Q# code samples
+    href: https://docs.microsoft.com/en-us/samples/browse/?languages=qsharp
+- name: The QDK in-depth
   items:
   - name: Quantum development techniques
     topicHref: techniques/index.md
     href: techniques/toc.yml
   - name: Q# programming language
     topicHref: language/index.md
     href: language/toc.yml
-  - name: Q# libraries
-    href: libraries/toc.yml
   - name: Quantum simulators and resource estimators
     href: machines/toc.yml
-  - name: Quantum computing concepts
-    href: concepts/toc.yml
-- name: How-to guides
-  items:
-  - name: Create a Q# project
-    href: how-to-guides/create-qsharp-project.md
-- name: Q# Learning Resources
-  items:
-  - name: Learn with the Quantum Katas
-    href: intro-to-katas.md
-  - name: Q# Samples
-    href: https://docs.microsoft.com/en-us/samples/browse/?languages=qsharp
-- name: Resources
-  items:
-  - name: Glossary
-    href: Glossary.md
-  - name: Further reading
-    href: for-more-info.md
-  - name: QDK Release Notes
-    topicHref: relnotes/index.md
-    href: relnotes/toc.yml
-  - name: Contributing to the QDK
-    href: contributing/toc.yml
-    topicHref: contributing/index.md
-  - name: License
-    href: license.md
+- name: Q# libraries
+  href: libraries/toc.yml
+- name: Theoretical concepts
```

Comment:
I like the idea here, but "theoretical" could be a bit off-putting to some readers perhaps? It feels like that communicates that the Quantum Development Kit is more of a theoretical thing than something that exists and that you can go use today. To be fair, I also don't know of a better word to use here, sorry.

Reply:
This one is trying to distinguish QDK (the rest of the docs) from the math and physics theories behind quantum computing. Maybe we call it "Math of Quantum Computing", "Theory of Quantum Computing"?

Reply:
Personally I thought the original "Quantum computing concepts" was fine, especially now that we've disambiguated out of the overarching "Concepts" section. Submitting a PR now, will include this change back--let me know what you think